docs/configuration.md

```diff
@@ -19,7 +19,7 @@ For example, the config object may look similar to the following:
 | Field Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | allowCookies | Boolean | `false` | Enable the web client to set and read two cookies: a session cookie named `cwr_s` and a user cookie named `cwr_u`.<br/><br/>`cwr_s` stores session data including an anonymous session ID (uuid v4) created by the web client. This allows CloudWatch RUM to compute sessionized metrics like errors per session.<br/><br/>`cwr_u` stores an anonymous user ID (uuid v4) created by the web client. This allows CloudWatch RUM to count return visitors.<br/><br/>`true`: the web client will use cookies<br/>`false`: the web client will not use cookies. |
-| aws:releaseId | string | `undefined` | The application release id used to find a matching source map, if any. Must match regex `^[a-zA-Z0-9_\-:/\.]{1,200}$`, including size limit 200 |
+| awsReleaseId | String | `undefined` | The application release id used to find a matching source map, if any. Must match regex `^[a-zA-Z0-9_\-:/\.]{1,200}$`, including size limit 200 |
```

Comment:
suggestions:
1. If we’re not using the colon, I don’t see a need to keep “aws” as a prefix. I think it’ll be cleaner to just do “releaseId”. 
2. “The releaseId will be used to retrieve source map(s), if any, when RUM service unminifies JavaScript error stack traces. It should be unique to each application release and match regex `^[a-zA-Z0-9_\-:/\.]{1,200}$`, including size limit of 200. ”